tests/lib/Reference/WorkPackageReferenceProviderTest.php

```diff
@@ -43,23 +43,23 @@ protected function setUp(): void {
 	/**
 	 *
 	 * @param array<string> $onlyMethods
-	 * @param null $configMock
-	 * @param null $iL10N
-	 * @param null $iURLGenerator
-	 * @param null $refrenceMangager
-	 * @param null $openProjectAPIService
-	 * @param null $userId
+	 * @param MockObject|null $configMock
+	 * @param IL10N|null $iL10N
+	 * @param IURLGenerator|null $iURLGenerator
+	 * @param ReferenceManager|null $refrenceMangager
+	 * @param OpenProjectAPIService|null $openProjectAPIService
+	 * @param string|null $userId
 	 *
-	 * @return MockObject
+	 * @return WorkPackageReferenceProvider|MockObject
 	 */
 	public function getWorkReferenceProviderMock(
-		array $onlyMethods = ['request'],
-		$configMock = null,
-		$iL10N = null,
-		$iURLGenerator = null,
-		$refrenceMangager = null,
-		$openProjectAPIService = null,
-		$userId = null
+		array                 $onlyMethods = ['request'],
+		MockObject            $configMock = null,
```

Comment:
I think we should be able to do 
```suggestion
		?MockObject            $configMock = null,
```
and then we can even delete the PHPDoc section and don't have to maintain both

Reply:
Good to have PHPDoc but instead i cleared up the Type in parameters.

Reply:
I have created a `workpackage` related to updating the code according to php 8.0. This can be done as house keeping work.